PYME/localization/FitFactories/__init__.py

```diff
@@ -22,8 +22,13 @@
 ################
 import glob
 import os
+from PYME import config
+import importlib
 
-__all__ = [os.path.splitext(os.path.split(p)[-1])[0] for p in glob.glob(__path__[0] + '/[a-zA-Z]*.py')]
+# discover PYME-internal fitfactories and fully qualify them
+__all__ = ['PYME.localization.FitFactories.' + os.path.splitext(os.path.split(p)[-1])[0] for p in glob.glob(__path__[0] + '/[a-zA-Z]*.py')]
```

Comment:
`__all__` is a special variable (although this looks like it's my mistake, just propagated)